src/prefect/cli/task_run.py

```diff
@@ -0,0 +1,223 @@
+"""
+Command line interface for working with task runs
+"""
+
+import logging
+from typing import List
+from uuid import UUID
+
+import httpx
+import pendulum
+import typer
+from prefect._vendor.starlette import status
+from rich.pretty import Pretty
+from rich.table import Table
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli._utilities import exit_with_error
+from prefect.cli.root import app
+from prefect.client.orchestration import get_client
+from prefect.client.schemas.filters import LogFilter, TaskRunFilter
+from prefect.client.schemas.objects import StateType
+from prefect.client.schemas.sorting import LogSort, TaskRunSort
+from prefect.exceptions import ObjectNotFound
+
+task_run_app = PrefectTyper(
+    name="task-run", help="Commands for interacting with task runs."
+)
+app.add_typer(task_run_app, aliases=["task-runs"])
+
+LOGS_DEFAULT_PAGE_SIZE = 200
+LOGS_WITH_LIMIT_FLAG_DEFAULT_NUM_LOGS = 20
+
+
+@task_run_app.command()
+async def inspect(id: UUID):
+    """
+    View details about a task run.
+    """
+    async with get_client() as client:
+        try:
+            task_run = await client.read_task_run(id)
+        except httpx.HTTPStatusError as exc:
+            if exc.response.status_code == status.HTTP_404_NOT_FOUND:
+                exit_with_error(f"Task run {id!r} not found!")
+            else:
+                raise
+
+    app.console.print(Pretty(task_run))
+
+
+@task_run_app.command()
+async def ls(
+    task_name: List[str] = typer.Option(None, help="Name of the task"),
```

Comment:
How about `task_run_name` here?

Reply:
Thank you , I have updated as per your comments